system-addon/content-src/components/Card/_Card.scss

```diff
@@ -59,7 +62,7 @@
     &::after {
       border-bottom: 1px solid rgba(0, 0, 0, 0.05);
       bottom: 0;
-      content: " ";
+      content: ' ';
```

Comment:
Any idea if having `content: ''` vs `content: ' '` makes any differences in the places we're touching here?

Reply:
That's a good question - I'm now wondering that myself.

I assume an empty set of single quotes would render no characters at all and for the latter, a single white space ([ASCII value of 32](https://ascii.cl/)).

Question that needs to be asked now is if we need that single white spaces and what purpose does it serve?